### DISCUSSION TITLE
companion not detected - snap Chromium on Linux

### DISCUSSION THREAD

**Discussion by mtosques** [aclap-dev/video-downloadhelper#1513] [Help and Q&A]
### Your question

Hi, 
companion is not detected from Snap Chromium.
I'm on the Ubuntu 24.10.
Firefox (no Snap) works perfectly.

already tried re-registering.
Diagnostic output:

  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/home/mt/.local/share/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/mt",
  "converterBinary": "/home/mt/.local/share/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360"

Thanks

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"error":"Specified native messaging host not found.","found":false}
license: {"unneeded":true}
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
{
    "used_history_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 3
}

**Comment by Mattmjs** [comment]
Have you read and followed the directions from the Wiki
https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized#ubuntu-and-flatpak--snap-based-systems
